openseeds/ols-5/cohort-notes/week-06.md

```diff
@@ -0,0 +1,337 @@
+
```

Comment:
The cohort notes should go in the OLS-5 GitHub repo (https://github.com/open-life-science/ols-5) as we do for the other cohorts

Reply:
A [PR](https://github.com/open-life-science/ols-5/pull/36) adding the `notes` and `emails` has been made in the OLS-5 repo. 🙏